templates/terraform/custom_delete/noop_on_destroy_appengine_version.go.erb

```diff
@@ -0,0 +1,48 @@
+if d.Get("noop_on_destroy") == false {	
```

Comment:
nit: It's a little clearer if you return immediately when the value is true instead of entering a long block when the value is false.